test/api/controllers/fda_applications.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const FdaApplication = require('../../../api/models/fda_application');
+
+describe('FdaApplication', () => {
+  before(clearDB);
+
+  afterEach(clearDB);
+
+  describe('GET /v1/fda_applications/{id}', () => {
+    it('returns 404 if there\'s no FDA application with the received ID', () => (
+      server.inject('/v1/fda_applications/BADFDAID')
+        .then((response) => {
+          response.statusCode.should.equal(404);
+        })
+    ));
+
+    it('returns the FDA application', () => (
+      factory.create('fda_application').then((model) => {
+        return server.inject('/v1/fda_applications/' + model.attributes.id)
+          .then((response) => {
+            response.statusCode.should.equal(200);
+
+            const expectedResult = JSON.parse(JSON.stringify(model.toJSON()));
+            const result = JSON.parse(response.result);
```

Comment:
There's a global method (ugly, I know :sad:) named `toJSON()` that you can use here, as `const expectedResult = toJSON(model)` (I think you don't need to do `toJSON(model.toJSON())`, but I haven't tested)

Also, I don't think you need to parse the `response.result`, as it should already be handled by Hapi.